frontend/views/components/PageSection.vue

```diff
@@ -1,7 +1,6 @@
 <template lang="pug">
 section.card
-  h3.title(v-if='title')
-    i18n {{title}}
```

Comment:
I believe we should keep i18n here.

Reply:
No, the title should be translated already